packages/vue/__tests__/svgNamespace.spec.ts

```diff
@@ -54,7 +55,8 @@ describe('SVG support', () => {
 
     // set a transition class on the <div> - which is only respected on non-svg
     // patches
-    ;(f2 as any)._vtc = ['baz']
+    ;(f2 as any)[vtcKey] = ['baz']
+    console.log('我打印了f2', f2)
```

Comment:
Please remove console for debugging.

Reply:
No problem. I found unused `@ts-expect-error` directives in `defineComponent.test-d.ts`, and these caused errors during the commit process. Therefore, I deleted them.